contracts/contracts/sbtc-withdrawal.clar

```diff
@@ -172,14 +183,14 @@
       )
 
       ;; Call into registry to confirm accepted withdrawal
-      (try! (contract-call? .sbtc-registry complete-withdrawal-accept request-id bitcoin-txid output-index signer-bitmap fee))
+      (try! (contract-call? .sbtc-registry complete-withdrawal-accept request-id bitcoin-txid output-index signer-bitmap fee burn-hash burn-height))
 
       (ok true)
   )
 )
 
 ;; Reject a withdrawal request
-(define-public (reject-withdrawal-request (request-id uint) (signer-bitmap uint))
+(define-public (reject-withdrawal-request (request-id uint) (signer-bitmap uint) (burn-hash (buff 32)) (burn-height uint))
```

Comment:
My bad, when I asked for these fields for withdrawals I meant for withdrawal accept only. We don't need to include this for rejects as well.

Reply:
Updated